docs/overviews/overview-publishers.md

```diff
@@ -48,6 +48,8 @@ The following documentation resources are available for publishers to implement
 | Mobile: iOS  | [UID2 SDK for iOS Reference Guide](../sdks/uid2-sdk-ref-ios.md) | This integration guide is for publishers who want to establish client identity using UID2 and retrieve advertising tokens on iOS devices. | Publishers |
 | Mobile: Android  | [UID2 SDK for Android Reference Guide](../sdks/uid2-sdk-ref-android.md) |  This integration guide is for publishers who want to establish client identity using UID2 and retrieve advertising tokens on Android devices. | Publishers |
 | Publisher/SSP Integration with GAM | [Publisher - Google Ad Manager Secure Signals](../guides/google-ss-integration.md) | This integration guide covers the additional steps needed for publishers using UID2 with the Google Ad Manager Secure Signals feature (previously known as Encrypted Signals for Publishers, ESP). | Publishers |
```

Comment:
we should put this above Mobile iOS row so all mobile+GMA plugin stuff together.

_Publisher/SSP Integration with GAM_ is more relevant to JS integration

Actually could we split this big tables into 3 smaller tables - 

1. web integration
2. mobile and Google GMA/IMA integration 
3. Operator Deployment (90%+ publishers wouldn't care about this section i would say so is best separating this away from the rest)

Reply:
Updated. Please check carefully -- particularly 1) the breakdown, 2) intro wording.

Reply:
@sunnywu : done but hoping you'd take a look at the update to be sure I have the breakdown correct, plus the intro text.

Reply:
looks good to me!